server/monitor/src/main/appended-resources/META-INF/LICENSE

```diff
@@ -75,14 +75,23 @@ Files:
      * Dual licensed under the MIT and GPL licenses.
      * http://benalman.com/about/license/
 
-## Bootstrap v5.3.1 (https://getbootstrap.com/)
+## Bootstrap v5.3.2 (https://getbootstrap.com/)
```

Comment:
I must be going crazy, because I thought I had checked these already. But nice catch.

Reply:
I think we had discussed who would fix this on the bootstrap ticket but neither of us ever got around to saying that we would do it.

Reply:
I definitely remember checking. I definitely re-downloaded the 5.3.2 versions, to make sure they matched bit-for-bit. I thought I checked the version in the LICENSE files at the same time, but maybe I saw 5.3.1 and my brain just replaced the 1 with a 2 in my head. :shrug: